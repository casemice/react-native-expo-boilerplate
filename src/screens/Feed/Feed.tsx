import React from 'react';
import { Text, View } from 'react-native';
import FeedStyles from './Feed.style';

const Feed: React.FunctionComponent = () => {
  return (
    <View style={FeedStyles.container}>
      <Text>Feed Screen</Text>
    </View>
  );
};

export default Feed;
