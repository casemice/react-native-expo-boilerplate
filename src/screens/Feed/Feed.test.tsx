import React from 'react';
import renderer from 'react-test-renderer';

import Feed from './Feed';

describe('Feed Screen', () => {
  it('Renders as expected', () => {
    const tree = renderer.create(<Feed />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
