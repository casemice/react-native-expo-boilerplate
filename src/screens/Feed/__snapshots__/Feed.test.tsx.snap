// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed Screen Renders as expected 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Feed Screen
  </Text>
</View>
`;
